bin/common/create_kubedns.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+
+echo "Waiting for kubectl to respond..."
+while ! kubectl get pods --all-namespaces; do
```

Comment:
Probably if it takes a long time. I typically see `==> default: No resources found.` once. I'll silence this